umpire-config.cmake.in

```diff
@@ -7,6 +7,11 @@
 get_filename_component(UMPIRE_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
 set(UMPIRE_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/include")
 
+if (NOT DEFINED camp_DIR)
```

Comment:
I think this whole thing should be inside a `if (NOT TARGET camp)` block.

Reply:
Good catch!